pkg/virt-controller/watch/vm/vm.go

```diff
@@ -517,6 +517,14 @@ func (c *Controller) handleDataVolumes(vm *virtv1.VirtualMachine) (bool, error)
 		} else if curDataVolume.Status.Phase != cdiv1.Succeeded &&
 			curDataVolume.Status.Phase != cdiv1.WaitForFirstConsumer &&
 			curDataVolume.Status.Phase != cdiv1.PendingPopulation {
+
+			if curDataVolume.Status.Phase == cdiv1.Pending {
```

Comment:
I think this if should be after ready=false next to the other status.phase check
Also might worth to also omit an event like we do for the failure status case